src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/JSON1006.ValueTooLarge_LocationId_Invalid.sarif

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing",
+          "rules": [
+            {
+              "id": "JSON1006",
```

Comment:
>JSON1006 [](http://example.com/codeflow?start=21&length=8)

The rule factoring here is not wise. Why is it important to have one rule to detect something is too large and another rule for something that is too small?  Does this really help the user? Would they, for example, need two different help topics to clarify this distinction?

We should have created something like 'ValueOutOfRange' and collapsed all this analysis into it. 

Please open an issue for this improvement.

Reply:
agree, created an issue in jschema repo:
[https://github.com/microsoft/jschema/issues/142](https://github.com/microsoft/jschema/issues/142)